demo/index.html

```diff
@@ -41,100 +46,116 @@
-            <a class="u-fg-inherit" href="menus">Menus</a>
-            <p class="u-zeta">@zendeskgarden/react-menus</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="modals">Modals</a>
-            <p class="u-zeta">@zendeskgarden/react-modals</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="notifications">Notifications</a>
-            <p class="u-zeta">@zendeskgarden/react-notifications</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="pagination">Pagination</a>
-            <p class="u-zeta">@zendeskgarden/react-pagination</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="radios">Radios</a>
-            <p class="u-zeta">@zendeskgarden/react-radios</p>
-          </div>
-        </div><div class="l-grid__item u-1/4">
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="ranges">Ranges</a>
-            <p class="u-zeta">@zendeskgarden/react-ranges</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="select">Select</a>
-            <p class="u-zeta">@zendeskgarden/react-select</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="selection">Selection</a>
-            <p class="u-zeta">@zendeskgarden/react-selection</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="tabs">Tabs</a>
-            <p class="u-zeta">@zendeskgarden/react-tabs</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="tags">Tags</a>
-            <p class="u-zeta">@zendeskgarden/react-tags</p>
-          </div>
-        </div><div class="l-grid__item u-1/4">
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="textfields">Textfields</a>
-            <p class="u-zeta">@zendeskgarden/react-textfields</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="theming">Theming</a>
-            <p class="u-zeta">@zendeskgarden/react-theming</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="toggles">Toggles</a>
-            <p class="u-zeta">@zendeskgarden/react-toggles</p>
-          </div>
-          <div class="u-mb">
-            <a class="u-fg-inherit" href="tooltips">Tooltips</a>
-            <p class="u-zeta">@zendeskgarden/react-tooltips</p>
+    <div class="container-fluid">
+      <h1 class="c-main__title">React Components</h1>
+      <p class="c-main__subtitle u-delta u-mt-lg" style="white-space: normal;">
+        This documentation includes React components and utilities that provide visuals, keyboard-navigation, localization, and accessibility
+        defined within the Garden Design System.
+      </p>
+      <div class="c-main__body" style="margin-top: 0; margin-left: 16px; margin-right: 16px;">
```

Comment:
I'd recommend removing the `style` attribute and add `l-wrapper-720` so that this page falls in line with `css-components`.

Reply:
This is now update to match css-components.

Originally had the custom styling since we were listing the package names, but it looks a lot better with the padding now that we are only showing the section titles.